addons/web/static/src/views/graph/graph_view.js

```diff
@@ -17,37 +17,31 @@ const viewRegistry = registry.category("views");
 
 const { Component } = owl;
 
-const KEYS = [
-    "additionalMeasures",
-    "disableLinking",
-    "display",
-    "fields",
-    "fieldAttrs",
-    "groupBy",
-    "measure",
-    "mode",
-    "order",
-    "resModel",
-    "stacked",
-    "title",
-];
-
 export class GraphView extends Component {
     setup() {
         this.actionService = useService("action");
 
         let modelParams;
-        if (this.props.state && this.props.state.metaData) {
-            // check this (dashboard testss)
+        if (this.props.state) {
             modelParams = this.props.state.metaData;
         } else {
             const { arch, fields } = this.props;
             const parser = new this.constructor.ArchParser();
             const archInfo = parser.parse(arch, fields);
-            modelParams = {};
-            for (const key of KEYS) {
-                modelParams[key] = key in archInfo ? archInfo[key] : this.props[key];
-            }
+            modelParams = {
+                additionalMeasures: this.props.additionalMeasures,
+                disableLinking: Boolean(archInfo.disableLinking),
+                displayScaleLabels: this.props.displayScaleLabels,
+                fieldAttrs: archInfo.fieldAttrs,
+                fields: this.props.fields,
+                groupBy: archInfo.groupBy,
```

Comment:
maybe avoid to set default in arch processor (du coup) and do what you do for stacked here?